joomla/joomla-cms
# Custom Fields - display units

It is very common to need to display a unit of measurement with a field. Currently there is no way to do this without creating custom layouts for every field and every unit when needed.

This PR adds a new param to every custom field "Suffix" which allows you to add a unit of measurement (or any other text) to be rendered after the value of the field.

The new field is disabled by default so three is no b/c issue

### To test
Apply the pr and then create a new custom field and you will see the new Suffix param in the render options.
![image](https://user-images.githubusercontent.com/1296369/50965217-462f5580-14c9-11e9-8159-3818d2fee39a.png)

## Example usage
### Before
![image](https://user-images.githubusercontent.com/1296369/50965194-39aafd00-14c9-11e9-87e6-5bff2c2d8167.png)

### After
![image](https://user-images.githubusercontent.com/1296369/50965166-1b450180-14c9-11e9-9b34-84b01b205c47.png)

##Plurals
If you need to have a different suffix for a plural then you can do this as follows
1. In the suffix field enter a key such as MINUTE_N
2. Create a language override  MINUTE_N = Minutes
3. Create any additional language overrides for specific numbers eg  MINUTE_N_1 = Minute

The output will now be 
0 Minutes
1 Minute
10 Minutes
